gomock_generator/gomockgenerator/generate.go

```diff
@@ -28,6 +28,8 @@ type GenerationConfiguration struct {
 	ConcurrentGoroutines int
 	// NoGoMod by default we'll consider go modules is enabled, mockgen will be called with -mod=mod to read interfaces in modules instead of default GOPATH
 	NoGoMod bool
+	// SourcePath override the output base path of the mocks
+	SourcePath string
```

Comment:
issue: I'm not sure I understand the choice of words here for the variable name. It mentions `Source` but it seems to be the location of what this program outputs. What do you think about `MocksPath` or `MocksRootPath`?

Reply:
I'm not precious about the naming... but it is the location of your codebase. MocksRootPath works. I'll update it.